config/models.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "gpt-4o",
+    "deployment_id": "23590sojsglsfsfsfdsjgls21592jfs"
```

Comment:
Let's not hardcode deployment_id in json and publish it into an open-source repo. I think we can load them as env variables, since that would be more secure and flexible approach. In this case we will also have all credentials for llms handled in a similar manner

Reply:
They will not be hard-coded, we will provide them as JSON in a k8s ConfigMap that is mounted to our application k8s deployment/pod. As we agreed before, I find this better as adding a new model doesn't need any extra env var and deleting doesn't need to delete env vars. I find this approach is more flexible than adding every model name and deployment ID as an env var.